Lab1 Implemented Fib

sirigithub opened the pull request:
Fixes #367 

Review of colleague's PR #

#### Changes proposed in this PR:
- Implemented  Lab1 Fibonacci numbers
- Tests for  Lab1

lexual-anz committed:
Implement fibonnaci printing with fib()

lexual-anz committed:
Implement negafibonnaci, and improve test suite

Commit:
Implement Lab1

Rename variables

Edit Test

nicholascross committed:
Lab 4 - Implement numeronyms

- Implement function that returns a slice of numeronyms for its input strings
- Create an executable to demonstrate
- Add unit tests

odrij committed:
Lab1 Fibonacci and Negafibonacci number

odrij committed:
Tests for fibonacci function

Commit:
Modify fibonacci loop

lexual-anz committed:
testing main() function to get 100% test coverage

lexual-anz committed:
Implement bubble and insertion sort

lexual-anz committed:
Use table-driven testing. Better comment.

In responde to code review, adopted table-driven testing.

Also made sure function comment, had function as 1st
word in comment.

sirigithub committed:
Fibonaci calculator for positive and negative numbers

sirigithub committed:
Fibonaci calculator for positive and negative numbers

sirishavadrevu committed:
Merge branch 'master' of https://github.com/sirigithub/go-course into vadrevusfib

jessodri committed:
Merge branch 'master' into master

jessodri committed:
Merge pull request #246 from jessodri/master

Lab1 Fibonacci and Negafibonacci number

nicholascross committed:
Merge branch 'master' into lab4

nicholascross committed:
Merge pull request #216 from nicholascross/lab4

Implement numeronyms

davidbroughsmyth committed:
Implement Fibonacci and Negafibonacci

davidbroughsmyth committed:
Merge pull request #313 from davidbroughsmyth/lab1

lab1 - Implement Fibonacci and Negafibonacci

willshen24 committed:
Implemented letters lab with test cases

- Solution check for existence first and then increment it in map
- Sort Letter is done by using sort.string()
- Test cases covers all different unicodes and empty inputs

willshen24 committed:
Changes added after PR review:
-use testify testing library
-removed branching logic for letter mapping
-use better slice declaration for performance tuning

n0npax committed:
Complete lab06

Implement CRUD for pet structure
Implement Sync and mem storer
Implement tests

lexual committed:
Merge branch 'master' into lab02

lexual committed:
Merge pull request #261 from lexual/lab02

Implement bubble and insertion sort

willshen8 committed:
Merge branch 'master' into lab03

n0npax commented on 01_fib/sirigithub/main.go:
```diff
@@ -0,0 +1,44 @@
+	return x
+}
+
+// fibonaci series
```
Comments in go should start with function name. in this case `fib`

n0npax commented on 01_fib/sirigithub/main.go:
```diff
@@ -0,0 +1,44 @@
+var out io.Writer = os.Stdout
+
+//function returns absolute value of an interger
+func abs(x int) int {
```
the function can be taken from math lib. You're using it just once so casting to `int` should not be very painful. Can stay as is if you prefer.

n0npax reviewed (commented):
Please fix comments and I'll approve

willshen8 committed:
Merge pull request #333 from willshen8/lab03

Lab03 - Letters Frequency

camscale committed:
travis: Change golangci-lint download script

Travis is currently failing when running the script to download
golangci-lint, with little output to help show what's going wrong:

$ curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.16.0
golangci/golangci-lint info checking GitHub for tag 'v1.16.0'
golangci/golangci-lint info found version: 1.16.0 for v1.16.0/linux/amd64
The command "curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.16.0" failed and exited with 1 during .

Pull the script directly from golangci-lint's github repo instead of
using install.goreleaser.com, which seems to be breaking the download
script.

lexual committed:
Merge branch 'master' into lab1

camscale committed:
Merge pull request #372 from anz-bank/enable-travis-ci-download-debug

travis: Change golangci-lint download script

lexual committed:
Merge branch 'master' into lab1

lexual committed:
Merge pull request #148 from lexual/lab1

Lab1

Commit:
Implement Fibonaci and NegaFibinacci (#179)

* Implement recursion based function to return a slice of
  Fibonacci numbers starting at 0
* Implement functions to present fibonacci as a string / stdout

n0npax committed:
Impelent CR fixes

Improved error messages
Optimalize tests
Improve syntax
Add comments to exported types

brvishnu92 committed:
Merge branch 'master' into lab1

n0npax committed:
Merge branch 'master' into lab06

n0npax committed:
Merge pull request #358 from n0npax/lab06

Complete lab06

brvishnu92 committed:
Merge branch 'master' into lab1

brvishnu92 committed:
Merge pull request #174 from brvishnu92/lab1

Implement Lab1

sirishavadrevu committed:
Implemented review comments

sirishavadrevu committed:
fixed merge conflicts
